ksa_compliance/patches/_2024_08_19_update_old_fatoora_url_in_business_settings.py

```diff
@@ -0,0 +1,32 @@
+import frappe
+
+
+def execute():
+    print("Start updating old fatoora url")
+    sql = """
+            SELECT company, fatoora_server_url
```

Comment:
Instead of updating by `company`, it's clearer and less error-prone to update by `name` (ID). The current logic assumes company is unique, which is true but in an indirect way--we don't have a unique constraint, we generate a name that includes the company, so settings for the same company will generate the same (duplicate) name, violating the primary key constraint.